### PR TITLE
Remove legacy code from HdpiCanvas

### PR DESCRIPTION
**Pull request by iMoses**
Talked to Sean and got the green light to remove the `measureText` legacy polyfill

### CODE REVIEW

File: packages/ag-charts-community/src/util/dom.ts [hunk 1]
```diff
@@ -1,4 +1,48 @@
...
+    element.href = dataUrl;
+    element.download = fileName;
+    element.style.display = 'none';
+    body.appendChild(element);
+    element.click();
+    setTimeout(() => body.removeChild(element));
+}
+
+export function injectStyle(cssStyle: string) {
+    const document = getDocument();
```
Comment (alantreadway) [inline packages/ag-charts-community/src/util/dom.ts:45]: This won't work for server-side rendering (e.g. thumbnail rendering), we need a non-global `Document` in those cases.

File: packages/ag-charts-community/src/util/draw-utils.ts
Comment (alantreadway) [inline packages/ag-charts-community/src/util/draw-utils.ts:1]: Duplicate?

**Review by alantreadway — CHANGES_REQUESTED** [review]
Generally LGTM, but we need to allow `document` and `window` references to be passed in for server-side rendering cases.

File: packages/ag-charts-community/src/util/draw-utils.ts
Comment (iMoses) [reply packages/ag-charts-community/src/util/draw-utils.ts:1]: yeah, oops :)

File: packages/ag-charts-community/src/util/dom.ts (hunk 1 above)
Comment (alantreadway) [reply packages/ag-charts-community/src/util/dom.ts:45]: Discussed offline, we'll run with the changes and see the repurcussions.

**Review by alantreadway — APPROVED** [review]
LGTM

**Pull request merged**